controllers/topics/inviteMembers.js

```diff
@@ -1,11 +1,29 @@
-const {Topics, User, FcmToken} = require('../../databases/models');
+const {StreamChat} = require('stream-chat');
+const {CHANNEL_TYPE_STRING} = require('../../helpers/constants');
+const {Topics, User, FcmToken, TopicInvitations} = require('../../databases/models');
 const {messaging} = require('firebase-admin');
+const {v4: uuid} = require('uuid');
 
 module.exports = async (req, res) => {
   try {
     let response;
     let {member_ids, topic_id} = req.body;
 
+    let inviter = await User.findOne({
+      where: {
+        user_id: req?.userId
+      }
+    });
+
+    if (!inviter) {
+      response = {
+        success: false,
+        message: `Inviter user id : ${inviter} not found`
+      };
+
+      return res.status(400).json(response);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code looks good, but there's a minor issue. In line 21, you're trying to print the `inviter` object in the error message, which will result in `[object Object]`. You should print `req.userId` instead.
```diff
-         message: `Inviter user id : ${inviter} not found`
+         message: `Inviter user id : ${req.userId} not found`
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->